distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response.enso

```diff
@@ -52,6 +66,14 @@ type Response
         content_type_optional = self.internal_http_response.headers.firstValue "Content-Type"
         if content_type_optional.isPresent then content_type_optional.get else Nothing
 
+    ## Get the response content length in bytes.
```

Comment:
```suggestion
    ## Get the response content length in bytes. This method uses the `Content-Length` header, and does not read the input stream.
```

Reply:
I think it would be worth to stress in the doc (I know signature has it but that is less visible) that it may return Nothing if that header is not defined (which I think is not that uncommon).